offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -53,10 +51,12 @@ class OfferToPDFConverter implements OfferExporter {
 
     private final Path newOfferStyle
 
+    private final Path newOfferPrintStyle
+
+    private static final URI OFFER_HTML_TEMPLATE_URI = OfferToPDFConverter.class.getClassLoader()
+            .getResource("offer-template/offer.html").toURI()
     private static final Path OFFER_HTML_TEMPLATE =
-            Paths.get(OfferToPDFConverter.class.getClassLoader()
-                    .getResource("offer-template/offer.html")
-                    .toURI())
+            Paths.get(OFFER_HTML_TEMPLATE_URI)
     private static final Path OFFER_HEADER_IMAGE =
             Paths.get(OfferToPDFConverter.class.getClassLoader()
                     .getResource("offer-template/offer_header.png")
```

Comment:
Why not also adjusting it here as you have done above? I think it would make sense to fetch the uri first and then get the path either for all of the documents or not to do it like this. (have a consistent way, at least in one class)

Reply:
In this case, I need the URI to create a static template. For the resource files, it does not make sense to separately construct the URI right now. I removed the unused `Offer_HTML_TEMPLATE` constant.